sql/mozfun/norm/browser_version_info/udf.sql

```diff
@@ -22,7 +26,8 @@ WITH browser_info AS (
 SELECT
   assert.equals('45.9.0', browser_info.version),
   assert.equals(45, browser_info.major_version),
-  assert.equals(45.9, browser_info.minor_version),
+  assert.equals(.9, browser_info.minor_version),
```

Comment:
This is a significant change for how `minor_version` works and seems likely to affect downstream dashboards/analyses.

Reply:
I might suggest:
- Keep `minor_version` using `truncate_version()`, and add a separate field with the new behavior (e.g. `minor_revision`).
- Have `patch_version` be a string in `<major>.<minor>.<patch>` format for consistency with `major_version` and `minor_version`, and add a separate field with just the new numeric `.<patch>` value (e.g. `patch_revision`).

Reply:
Or I see from another comment that `browser_version_info` was only added recently, so maybe not much will break?

I do like the consistency of the new behavior, having all the parts numeric.

Reply:
Yeah some good points here, I initially thought this was worth the breaking change with some due diligence. I'll go ahead and call the patch: `patch_revision` and remove `patch_version` so no-one expects it to be a full version number.  --  If we do that we won't have to introduce a `patch_version` field at all (`.version` already contains the original version string)

Reply:
That sounds reasonable, though `patch_version` might still be useful in certain scenarios, as the full `.version` string might contain more than just the numeric parts `<major>.<minor>.<patch>` (e.g. `96.0.2b1`).